src/styles/blocks/card.scss

```diff
@@ -0,0 +1,90 @@
+.card {
+  width: 200px;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+
+  &__image {
+    margin: 0 auto;
+    width: 160px;
+    height: 134px;
+    display: block;
+  }
+
+  &__description {
+    color: $main-text-color;
+    margin-top: 40px;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    height: 36px;
```

Comment:
The height of the text blocks should not be set manually. The height should be defined by its content and padding if needed. Setting the height manually may lead to visual issues if the content exceeds the set height.